library/translation.inc.php

```diff
@@ -263,6 +263,15 @@ function html_header_show() {
     //  We commented this out since we are now standardizing encoding
     //  in the globals.php file.
```

Comment:
It is possible it will use another mechanism. Details of AM implementation and integration into OpenEMR are not defined much  this time :) What;s clear that it would replace all code that loads assets.

I would suggest to take a look on [Assetic](https://github.com/kriswallsmith/assetic) .